tests/experimental/wrappers/vector/test_normalize_observation.py

```diff
@@ -0,0 +1,51 @@
+"""Test suite for NormalizeObservationV0."""
+import numpy as np
+import pytest
+
+from gymnasium import spaces
+from gymnasium.vector import AsyncVectorEnv, SyncVectorEnv
+from gymnasium.wrappers.vector import NormalizeObservationV0
+from tests.testing_env import GenericTestEnv
+
+
+@pytest.mark.parametrize("class_", [AsyncVectorEnv, SyncVectorEnv])
+def test_normalize_obs(class_):
```

Comment:
Is it possible to test that the mean and std are very close for the standard wrapper and vector version after ~1000 steps? If there is a known seed this could help

This will actually test that the vector wrapper works very similarly to the normal wrapper